app/assets/stylesheets/about.scss.erb

```diff
@@ -91,3 +120,133 @@
   }
 
 }
+
+/** NAVIGATION ARROWS */
+
+.nav-circlepop {
+  position: relative;
+}
+
+.nav-circlepop a {
+  margin: 0 30px;
+  width: 50px;
+  height: 50px;
+}
+
+.nav-circlepop a::before {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+  background: #fff;
+  content: '';
+  opacity: 0;
+  -webkit-transition: -webkit-transform 0.3s, opacity 0.3s;
+  transition: transform 0.3s, opacity 0.3s;
+  -webkit-transform: scale(0.9);
+  transform: scale(0.9);
```

Comment:
Check out http://bourbon.io/docs/#transitions and http://bourbon.io/docs/#transform :)